### PR TITLE
Document how to run it

### LINKED ISSUE #2: Document how to run it
It is important do document the steps to run and test the environment so every person on the team can also test it in their own environments.

## Steps

- [ ] Update the README.md file with documentation on how to run

### PR DESCRIPTION
**Pull request by MelvinPariyadan**
Closes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
 # API
+
+We will use Node.js Express with MongoDB Architecture to make login and signup
+
+The backend will provide these APIs:
+
+Methods Urls Actions
+POST /api/auth/signup signup new account
+POST /api/auth/signin login an account
+POST /api/auth/signout logout the account
```
Comment (LBeghini) [inline README.md]:
```suggestion

```

File: README.md [hunk 2]
```diff
@@ -1 +1,70 @@
...
+
+## requirements
+
+Express 4.17.1
+cookie-session 1.4.0
+bcryptjs 2.4.3
+jsonwebtoken 8.5.1
+mongoose 5.13.13
+MongoDB
```
Comment (LBeghini) [inline README.md]:
```suggestion
- Express 4.17.1
- cookie-session 1.4.0
- bcryptjs 2.4.3
- jsonwebtoken 8.5.1
- mongoose 5.13.13
- MongoDB
```

File: README.md [hunk 3]
```diff
@@ -1 +1,70 @@
...
+
+# Initialize setup
+
+mkdir node-js-express-login--mongodb
```
Comment (LBeghini) [inline README.md]:
````suggestion

````

File: README.md [hunk 4]
```diff
@@ -1 +1,70 @@
...
+cd node-js-express-login--mongodb
```
Comment (LBeghini) [inline README.md]:
````suggestion

````

File: README.md [hunk 5]
```diff
@@ -1 +1,70 @@
...
+
+npm init
```
Comment (LBeghini) [inline README.md]:
```suggestion

```

File: README.md [hunk 6]
```diff
@@ -1 +1,70 @@
...
+npm install express mongoose cors cookie-session jsonwebtoken bcryptjs --save
```
Comment (LBeghini) [inline README.md]:
````suggestion
```shell
npm install  --save
````

File: README.md [hunk 7]
```diff
@@ -1 +1,70 @@
...
+// simple route
+app.get("/", (req, res) => {
+res.json({ message: "Welcome to bezkoder application." });
+});
+
+// set port, listen for requests
+const PORT = process.env.PORT || 8080;
+app.listen(PORT, () => {
+console.log(`Server is running on port ${PORT}.`);
+});
```
Comment (LBeghini) [inline README.md]:
```suggestion

```

File: README.md [hunk 8]
```diff
@@ -1 +1,70 @@
...
+
+node server.js
```
Comment (LBeghini) [inline README.md:23]:
````suggestion
```shell
node server.js
```
````

**Review by LBeghini — CHANGES_REQUESTED** [review]
Pretty obvious you just copy paste from the documentation, which is ok, but you also copied the commands to create the project again. What should be mentioned it is only the commands necessary to run the application.

**Review by SaraGrimaldi — APPROVED** [review]

File: README.md [hunk 9]
```diff
@@ -1 +1,26 @@
...
+- jsonwebtoken 8.5.1
+- mongoose 5.13.13
+- MongoDB
+
+# Initialize setup
+
+
+```shell
+npm install  --save
+
```
Comment (LBeghini) [inline README.md]:
````suggestion
```
````

**Review by LBeghini — CHANGES_REQUESTED** [review]

**Review by LBeghini — APPROVED** [review]

**Pull request merged**